nanome_vault/Vault.py

```diff
@@ -142,7 +149,10 @@ def get_server_url(self):
             s.close()
 
         if port != DEFAULT_WEB_PORT:
-            url += ':' + str(port)
+            if port == 443:
```

Comment:
Could we add another port variable, similar to `DEFAULT_WEB_PORT`? Maybe `SECURE_WEB_PORT` (or a better name)